### PR TITLE
Reproduce GWAS Tutorial

### PR DESCRIPTION
**Pull request by samuelmacedo83**

### CODE REVIEW

File: R/entries.R [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#' @export
+entries <- function(df){
```
Comment (javierluraschi) [inline R/entries.R]: I would rename to `hail_entries()`, I would also mention in the README that this function selects and explodes the data frame using `sparklyr.nested` and a link to the implementation. I think that would help users modify this if needed.

File: R/get_s.R [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#' @export
+get_s <- function(mt){
```
Comment (javierluraschi) [inline R/get_s.R]: I don't fully understand why the data frame created with `hail_dataframe()` does not contain the `s` identifier, so worth renaming to `hail_id()`. That said, I think in an ideal situation this gets extracted from `df` not the `mt`, but I might be missing something here. Do you have a better sense of why we need to extract the ideas from `mt`?

Comment (samuelmacedo83) [reply R/get_s.R]: In `matrixtable` we have entries, rows and cols. `s` is a col and have the same length than `entries`, but I'm not sure if it is in the same order to use `bind_cols`, for example.

**Comment by samuelmacedo83** [comment]
merged #4

**Pull request merged**